roles/lnls-ans-role-zabbix/vars/Debian-stretch.yml

```diff
@@ -1,5 +1,5 @@
 ---
 __zabbix_packages:
-  - zabbix-agent=1:4.0.3*
+  - "zabbix-agent={{ pk_version_zabbix_agent }}"
```

Comment:
Maybe a typo on `pk`?

Reply:
a typo consistent with variable definition in groups/all: passed test.
fixed